### PR TITLE
docs: slider, range slider, checkbox, and content

### LINKED ISSUE #297: pydocs for checkbox

### PR DESCRIPTION
**Pull request by ethanalvizo**
Closes #297, #298, #299, #440

### CODE REVIEW

**Comment by ethanalvizo** [comment]
@mofojed I commented out each prop that I omitted along with the reason they're not included. I thought this would be easier to see which things were left intentionally but if it's preferred to not see them at all I can change it

I also couldn't test the Content component properly. Wasn't sure how to pass in a child component into it.

File: plugins/ui/src/deephaven/ui/components/picker.py [hunk 1]
```diff
@@ -7,8 +7,16 @@
 from ..elements import BaseElement
 from .._internal.utils import create_props
 from ..types import ColumnName, Key
+from spectrum.layout import *
+from spectrum.validate import *
+from spectrum.events import *
```
Comment (mofojed) [inline plugins/ui/src/deephaven/ui/components/picker.py:12]: These are invalid imports. Should have the `.` in front of `spectrum` to indicate it's a relative directory:
```suggestion
from .spectrum.layout import *
from .spectrum.validate import *
from .spectrum.events import *
```

File: plugins/ui/src/deephaven/ui/components/spectrum/content.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    Number,
+    Position,
+    LabelPosition,
+)
+from .basic import spectrum_element
+from ...elements import Element
+
+
+def content(
+    *children: Any,
```
Comment (mofojed) [inline plugins/ui/src/deephaven/ui/components/spectrum/content.py:27]: I'm confused by your comment about not knowing how to pass a child component into `Content`. You pass it in just like a child to any other component (such as `Flex` or `View`).
```python
# basic content
my_content = ui.content("Hello world")

# With a Foo component
@ui.component
def Foo():
  value, set_value = ui.use_state(True)

  return ui.checkbox('Checked', value=value, on_change=set_value)

my_foo_content = ui.content(Foo(), UNSAFE_style={'background': 'red'})
my_foo_content2 = ui.content(Foo(), flex_grow=1, UNSAFE_style={'background': 'red'})
```

**Review by mofojed — CHANGES_REQUESTED** [review]
Need to fix imports, this does not install/run correctly and tests are not running as it is.

File: plugins/ui/src/deephaven/ui/components/spectrum/basic.py [hunk 3]
```diff
@@ -11,20 +11,12 @@ def spectrum_element(name: str, /, *children: Any, **props: Any) -> BaseElement:
     return BaseElement(f"deephaven.ui.spectrum.{name}", *children, **props)
 
 
-def checkbox(*children, **props):
+def contextual_help(*children, **props):
```
Comment (mofojed) [inline plugins/ui/src/deephaven/ui/components/spectrum/basic.py:14]: This isn't right - `contextual_help` was [already moved to it's own file](https://github.com/deephaven/deephaven-plugins/pull/480/files). Shouldn't be adding this back in to basic.py

File: plugins/ui/src/deephaven/ui/types/types.py [hunk 4]
```diff
@@ -43,6 +43,26 @@ class RowDataValue(CellData):
     """
 
 
+class SliderChange(TypedDict):
+    """
+    Data for a range slider change event.
+    { start: float, end: float }
```
Comment (mofojed) [inline plugins/ui/src/deephaven/ui/types/types.py:49]: This is redundant, given we already have the structure defined below.
```suggestion
```

**Review by mofojed — CHANGES_REQUESTED** [review]

**Review by mofojed — APPROVED** [review]

**Pull request merged**